manifests/n/Nvidia/Omniverse/1.9.8.2391/Nvidia.Omniverse.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Nvidia.Omniverse
+PackageVersion: 1.9.8.2391
+InstallerType: nullsoft
+ProductCode: ddd216ee-cf6c-55b0-9ca8-733b2ef622a0
+AppsAndFeaturesEntries:
+- DisplayVersion: 1.9.8
+Installers:
+- Architecture: x86

```

Comment:
```suggestion
- Architecture: x64
```

Validated online with [ThreadBook](https://s.threatbook.com/report/file/e6f257805089e383bc2128a7631ad4938756f8461e1fddc65c557f106b7f9f92), the installer wizard have to run in a 64-bit OS.

![image](https://github.com/microsoft/winget-pkgs/assets/49941141/2bf943a5-ba9c-4879-86d9-d6743a8b5dcd)